portfolio/src/main/webapp/index.html

```diff
@@ -28,6 +28,28 @@ <h2>My Background</h2>
           </p>
       <h2>My Pets</h2>
       <p>I love my pets and I hope you enjoy their portraits below:</p>
+      <div id="pets">
+          <div class="row">
+            <div class="column">
+                <img src="/images/Fern.jpeg" alt="Fern" style="width:100%">
+                </div>
```

Comment:
Each "</div>" should be aligned with the opening "<div ...>"

Reply:
Not enitrely sure what you mean, do not think most recent push encounters this problem.